programming-basics/alfabeto/README.md

```diff
@@ -0,0 +1,18 @@
+# Alfabeto
+
+Neste exercício é necessário verificar se uma dada `string` contém as 26 letras do alfabeto.
+
+Logo você deverá implementar a função `verificaAlfabeto`, que vai ter como entrada uma `string` de tamanho máximo de **100** letras e que deverá retornar como resposta um **booleano** `true` ou `false`, sendo:
+
+- `true` quando a `string` conter os 26 caracteres do alfabeto.
+- `false` quando a `string` não conter os 26 caracteres do alfabeto.
+
+## Requisitos
+
+Instalar as dependências do `programming-basics`, siga o tutorial [aqui](../README.md).
+
+## Testando
```

Comment:
We should unify test instructions and link it here to avoid repeat it for each exsercise